### PR TITLE
@orta=>Master settings view

### PR DESCRIPTION
**Pull request by sarahscott**

### CODE REVIEW

File: Classes/Controllers/Settings/ARLabSettingsMasterViewController.m [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
 
 @interface ARLabSettingsMasterViewController ()
-@property (weak, nonatomic) IBOutlet UIButton *settingsButton;
-@property (weak, nonatomic) IBOutlet UIButton *ogSettingsButton;
+@property (weak, nonatomic) IBOutlet UIButton *settingsIcon;
+@property (weak, nonatomic) IBOutlet UILabel *presentationModeLabel;
+
+@property (weak, nonatomic) IBOutlet ARLabSettingsSectionButton *syncContentButton;
+
+@property (weak, nonatomic) IBOutlet ARLabSettingsSectionButton *presentationModeButton;
```
Comment (sarahscott) [inline Classes/Controllers/Settings/ARLabSettingsMasterViewController.m]: really debated a uitableview, stackview, or some other organizational tool for these buttons but couldn't decide on one. what do you think?

File: Classes/Controllers/Settings/ARLabSettingsMasterViewController.m [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+@property (weak, nonatomic) IBOutlet UIView *presentationModeToggle;
+@property (weak, nonatomic) IBOutlet ARLabSettingsSectionButton *editPresentationModeButton;
+
+@property (weak, nonatomic) IBOutlet ARLabSettingsSectionButton *backgroundButton;
+@property (weak, nonatomic) IBOutlet ARLabSettingsSectionButton *emailButton;
+
+@property (weak, nonatomic) IBOutlet ARLabSettingsSectionButton *supportButton;
+@property (weak, nonatomic) IBOutlet ARLabSettingsSectionButton *logoutButton;
+
+@property (assign) BOOL presModeOn;
```
Comment (orta) [inline Classes/Controllers/Settings/ARLabSettingsMasterViewController.m]: perhaps `@property (assign, getter = isInPresentationMode) BOOL presentationMode` ?

File: Classes/Controllers/Settings/ARLabSettingsMasterViewController.m (hunk 1 above)
Comment (orta) [inline Classes/Controllers/Settings/ARLabSettingsMasterViewController.m]: You could use an [IBOutletCollection](http://useyourloaf.com/blog/interface-builder-outlet-collections.html) + an enum of all the button orders ( maybe even have the view setup logic done in a collection of classes ). I'd keep it as simple as possible like you're doing till it's obviously not working

**Comment by orta** [comment]
This is looking good, I'm happy to merge if you are :)

File: Classes/Controllers/Settings/ARLabSettingsMasterViewController.m (hunk 2 above)
Comment (sarahscott) [inline Classes/Controllers/Settings/ARLabSettingsMasterViewController.m]: yeah this will definitely be linked to a default once that's built

**Comment by sarahscott** [comment]
:ok_hand:

**Comment by orta** [comment]
Yep, add the CHANGELOG entry and we're good to go :+1:

**Pull request merged**